ios/AppLovinMAX.m

```diff
@@ -972,6 +972,17 @@ - (void)setAdViewPlacement:(nullable NSString *)placement forAdUnitIdentifier:(N
     });
 }
 
+- (void)setAdViewCustomData:(nullable NSString *)customData forAdUnitIdentifier:(NSString *)adUnitIdentifier adFormat:(MAAdFormat *)adFormat
+{
+    dispatch_async(dispatch_get_main_queue(), ^{
+
+        [self log: @"Setting customData \"%@\" for \"%@\" with ad unit identifier \"%@\"", customData, adFormat, adUnitIdentifier];
```

Comment:
I think we should spell it out (since for example we spell out "ad unit identifier" and this is for human readable logs).
```suggestion
        [self log: @"Setting custom data \"%@\" for \"%@\" with ad unit identifier \"%@\"", customData, adFormat, adUnitIdentifier];
```